packages/xchain-ethereum/__mocks__/responses/mimir.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
[suggestion] To have mocks of same Midgard responses more DRY, can you define it in `xchain-util` to use it everywhere?